mlir/test/Dialect/Index/inliner-interface.mlir

```diff
@@ -0,0 +1,14 @@
+// RUN: mlir-opt %s -inline | FileCheck %s
+
+// CHECK-LABEL: @main
+func.func @main(%arg0: i32) -> index {
+		// CHECK-NOT: call
+		// CHECK: index.castu
```

Comment:
Note: this check isn't very useful as is because it'll match the one in function @f 

You can force it to match in main by adding `// CHECK-LABEL: func @f(%arg0: i32)` below before @f